.github/workflows/prerelease.yml

```diff
@@ -47,202 +55,203 @@ jobs:
+      EXPORTER_CHANGELOG: ${{ steps.check.outputs.EXPORTER_CHANGELOG }}
 
-  build_push_linux_artifacts:
-    needs: [create_prerelease]
-    name: Push Linux Artifacts
+  build_linux_artifacts:
+    needs: [exporter_variables]
+    name: Build Linux artifacts and packages
     runs-on: ubuntu-latest
-    if: ${{ needs.create_prerelease.outputs.CREATE_RELEASE == 'true' && needs.create_prerelease.outputs.PACKAGE_LINUX == 'true'}}
+    if: ${{ needs.exporter_variables.outputs.CREATE_RELEASE == 'true' && needs.exporter_variables.outputs.PACKAGE_LINUX == 'true'}}
     steps:
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
         with:
-          go-version: ${{env.GO_VERSION}}
-      - name: Checkout code
-        uses: actions/checkout@v2
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: nri-config-generator/go.mod
+      - name: Install goreleaser
+        uses: goreleaser/goreleaser-action@v4
+        with:
+          version: v1.13.1
+          install-only: true
       - name: Install dependencies
         run: |
           sudo apt-get install rpm ruby ruby-dev rubygems build-essential rpm gnupg2 gpg-agent debsigs
           sudo gem install --no-document fpm
+
       - name: Load variables
         id: vars
         run : |
           export GOPATH=$(go env GOPATH)
-          export EXPORTER_PATH=${{ needs.create_prerelease.outputs.EXPORTER_PATH }}
+          export EXPORTER_PATH=${{ needs.exporter_variables.outputs.EXPORTER_PATH }}
           source ./scripts/common_functions.sh
           loadVariables $EXPORTER_PATH
           setStepOutput
-      - id: download_artifacts
-        name: Download stashed folders
-        uses: actions/download-artifact@v2
-        with:
-          name: SOURCE_DIR
-          path: exporters/${{ steps.vars.outputs.NAME }}/target/source
       - name: Package linux
         id: package
         env:
           GPG_MAIL: 'infrastructure-eng@newrelic.com'
           GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
-          GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+          GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }}  # base64 encoded
         run : |
-          make  build-${{ steps.vars.outputs.NAME }}
-          make  package-${{ steps.vars.outputs.NAME }}
+          make package-${{ steps.vars.outputs.NAME }}
       - name: Test package installability
         uses: newrelic/integrations-pkg-test-action/linux@v1
         with:
           tag: ${{ steps.vars.outputs.VERSION }}
           integration: nri-${{ steps.vars.outputs.NAME }}
           upgrade: false
           pkgDir: ./exporters/${{ steps.vars.outputs.NAME }}/target/packages/
-      - name: Upload Release Asset DEB
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload assets to be released
```

Comment:
are we no longer adding artifacts to the release itself?

Reply:
We are still adding the artifacts, it's done [here](https://github.com/newrelic/newrelic-prometheus-exporters-packages/blob/1c80fcc3732af03c2ad59aaac99445092c53dbd1/.github/workflows/prerelease.yml#L238) with the new action.

Reply:
oh, only at the end, even better, thanks! :)